### ISSUE TITLE
Fix module configuration for Angular 7.x

### ISSUE THREAD

**Issue by MagicCurlyHair** [angular-extensions/elements#27]
Currently, the application fails with the following error when using the module configuration:

**ERROR TypeError: Cannot read property 'replace' of undefined**

The following code produces the error:
```ts
const lazyElementConfig: LazyElementModuleOptions = {
    elementConfigs: [
        { tag: 'ba-layout', url: 'http://localhost:4000' }
    ]
};

@NgModule({
    declarations: [
        AppComponent
    ],
    imports: [
        BrowserModule,
        LazyElementsModule.forRoot(lazyElementConfig)
    ],
    bootstrap: [
        AppComponent
    ],
    schemas: [CUSTOM_ELEMENTS_SCHEMA]
})
export class AppModule {}
```
In AppComponent:
```ts
@Component({
    selector: 'app-root',
    template: `
        <ba-layout *axLazyElement>
        </ba-layout>
    `
})
export class AppComponent {}
```
I checked the source code and it seems that the directive doesn't look up the config on Angular 7.x branch. Could you please fix it? Or I could submit a PR myself.

**Comment by tomastrajan** [comment]
The solution to this problem is to backport new features into 7.x and make a npm release, will try to do this over the weekend :)

**Comment by MagicCurlyHair** [comment]
Sounds great, thank you!

**Comment by MagicCurlyHair** [comment]
I just tested this in another project that uses Angular 8 and this approach still doesn't work for me.
Here is the source code:
```ts
this.elementsLoaderService
      .loadElement(this.url, elementTag, this.isModule)
```
It seems that the URL is always resolved from the input of the directive, regardless of whether the config was passed or not.

**Comment by tomastrajan** [comment]
Fixed in `7.3.0`, enjoy!
